download.html

```diff
@@ -0,0 +1,51 @@
+<div xmlns="http://www.w3.org/1999/xhtml" id="download-modal" class="modal">
+
+  <style>
+    #progress-bar-container {
+      width: 100%;
+      background-color: #e9ecef;
+      border-radius: 5px;
+      height: 15px;
+      margin: 10px 0;
+      overflow: hidden;
+    }
+
+    #progress-bar {
+      height: 100%;
+      width: 0%;
+      background-color: #0d6efd;
+      text-align: center;
+      line-height: 15px;
+      color: white;
+      border-radius: 5px;
+      transition: width 0.3s ease;
+    }
+
+    #download-progress-label,
+    #current-file-label {
+      font-size: 14px;
+      margin: 5px 0;
+    }
+  </style>
+
+  <!-- Credits to https://www.w3schools.com/howto/howto_css_login_form.asp -->
+  <span class="close-modal close" title="Close Modal">&#215;</span>
+
+  <!-- Modal Content -->
+  <div class="modal-content animate">
+    <div class="container">
+      <h1>Download Files</h1>
+      <p id="download-progress-label">Preparing download...</p>
+
+      <div id="progress-bar-container">
+        <div id="progress-bar"></div>
+      </div>
+
+      <p id="current-file-label">Waiting...</p>
+    </div>
+  </div>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.0/jszip.min.js"></script>
+  <script type="text/javascript" src="../resources/js/download.js">
```

Comment:
I know other references to resources are made with "../" but I believe this is wrong and only works accidentally on the root level. The proper relative path is simply "resources/"
```suggestion
  <script type="text/javascript" src="resources/js/download.js">
```